src/dialogs/more-info/ha-more-info-logbook.ts

```diff
@@ -153,6 +153,11 @@ export class MoreInfoLogbook extends LitElement {
           max-height: 250px;
           overflow: auto;
         }
+        @media all and (max-width: 450px), all and (max-height: 500px) {
+         ha-logbook {
+           max-height: calc(100vh - 230px);
```

Comment:
The `230px` will only work for a timeline graph, if you have any other type of graph, or no history at all it will not work correctly.

You should work with a flex box to fix that.

Reply:
@bramkragten Thank you, didn't play out that scenario. Will look into this further!

Reply:
@bramkragten The max-height is now unset on smaller screens. I believe this may provide the optimal scrolling experience on mobile. Please let me know if you disagree and prefer the flex layout.

Thanks for taking the time!